README.md

```diff
@@ -188,6 +215,56 @@ Possible values of this variable:
 
 Note: If not set explicitly on registration, the display name is set to the hostname by default. It is not possible to automatically revert it to the hostname, but it can be set so manually.
 
+```yaml
+    rhc_insights:
+      file_redaction:
+        commands: []
+        files: []
+        components: []
+```
+
+Specify lists of commands, files, and components to obmit from output
+
+```yaml
+    rhc_insights:
+      file_redaction:
+        commands: []
+        files: []
+        components: []
+      file_content_redaction:
+        keywords: []
+        patterns: []
+        regex_patterns: []
+```
+
+These are optional.
+Specify lists of commands, files, components, keywords and patterns to obmit from output.
+NOTE: You cannot mix plain string matching and regular expression matching.
+For more informaition on this topic read: [YAML-style denylist configuration for Red Hat Insights Client](https://access.redhat.com/articles/4511681).
+
+```yaml
+    rhc_insights:
+      loglevel: DEFAULT
+```
+
+Configures the log level; valid options are DEBUG (the default), INFO, WARNING, ERROR, CRITICAL.
+
+```yaml
+    rhc_insights:
+      obfuscate: false
+```
+
+Configures IP address obfuscation; valid values are `false` (the default),
+and `false`.
+
+```yaml
+    rhc_insights:
+      obfuscate_hostname: false
+```
+
+Configures hostname obfuscation; valid values are `false` (the default),
+and `false`. Requires `obfuscate: true`.
```

Comment:
```suggestion
and `true`. Requires `obfuscate: true`.
```

Reply:
Fixed in commit [ff0199b](https://github.com/linux-system-roles/rhc/pull/177/commits/ff0199b1f95e5490a32844b1530f5b49f3a0167f)